packages/utils/src/cmds/schema/diff/utils.js

```diff
@@ -0,0 +1,30 @@
+const URL = require('url').URL
+
+function isUrl(url) {
+    try {
+        new URL(url)
+        return true
+    } catch (e) {
+        return false
+    }
+}
+
+function btoa(str) {
+    return Buffer.from(str).toString('base64')
+}
+
+function basicAuthHeader(user, pass) {
+    const b64 = btoa(`${user}:${pass}`)
+    return `Basic ${b64}`
+}
+
+function isAbsoluteUrl(url) {
+    return url.indexOf('http://') === 0 || url.indexOf('https://') === 0
```

Comment:
[`String.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) is a bit more contemporary than using `indexOf`.